contracts/roles/OwnerManager.sol

```diff
@@ -0,0 +1,112 @@
+        identityRegistry = token.getIdentityRegistry();
+        issuersRegistry = identityRegistry.getIssuersRegistry();
+        issuersRegistry.removeTrustedIssuer(index);
+    }
+
+    function callUpdateIssuerContract(uint index, IClaimIssuer _newTrustedIssuer, uint[] calldata claimTopics, IIdentity onchainID) external {
+        require(isIssuersRegistryManager(address(onchainID)) && onchainID.keyHasPurpose(keccak256(abi.encode(msg.sender)), 1), "Role: Sender is NOT IssuersRegistryManager");
+        identityRegistry = token.getIdentityRegistry();
+        issuersRegistry = identityRegistry.getIssuersRegistry();
+        issuersRegistry.updateIssuerContract(index, _newTrustedIssuer, claimTopics);
+    }
+
+    function callAddClaimTopic(uint256 claimTopic, IIdentity onchainID) external {
+        require(isClaimRegistryManager(address(onchainID)) && onchainID.keyHasPurpose(keccak256(abi.encode(msg.sender)), 1), "Role: Sender is NOT ClaimRegistryManager");
+        identityRegistry = token.getIdentityRegistry();
+        topicsRegistry = identityRegistry.getTopicsRegistry();
+        topicsRegistry.addClaimTopic(claimTopic);
+    }
+
+    function callRemoveClaimTopic(uint256 claimTopic, IIdentity onchainID) external {
+        require(isClaimRegistryManager(address(onchainID)) && onchainID.keyHasPurpose(keccak256(abi.encode(msg.sender)), 1), "Role: Sender is NOT ClaimRegistryManager");
+        identityRegistry = token.getIdentityRegistry();
+        topicsRegistry = identityRegistry.getTopicsRegistry();
+        topicsRegistry.removeClaimTopic(claimTopic);
+    }
+
+    function callTransferOwnershipOnTokenContract(address newOwner) external onlyAdmin {
+        token.transferOwnershipOnTokenContract(newOwner);
+    }
+
+    function callTransferOwnershipOnIdentityRegistryContract(address newOwner) external onlyAdmin {
+        identityRegistry = token.getIdentityRegistry();
+        identityRegistry.transferOwnershipOnIdentityRegistryContract(newOwner);
+    }
+
+    function callTransferOwnershipOnComplianceContract(address newOwner) external onlyAdmin {
+        compliance = token.getCompliance();
+        compliance.transferOwnershipOnComplianceContract(newOwner);
+    }
+
+    function callTransferOwnershipOnClaimTopicsRegistryContract(address newOwner) external onlyAdmin {
+        identityRegistry = token.getIdentityRegistry();
+        topicsRegistry = identityRegistry.getTopicsRegistry();
+        topicsRegistry.transferOwnershipOnClaimTopicsRegistryContract(newOwner);
+    }
+
+    function callTransferOwnershipOnIssuersRegistryContract(address newOwner) external onlyAdmin {
+        identityRegistry = token.getIdentityRegistry();
+        issuersRegistry = identityRegistry.getIssuersRegistry();
+        issuersRegistry.transferOwnershipOnIssuersRegistryContract(newOwner);
+    }
```

Comment:
a function to set agents on the smart contracts that are AgentRole is still missing, i will add it soon :-)